llvm/lib/CodeGen/WindowScheduler.cpp

```diff
@@ -232,8 +232,11 @@ bool WindowScheduler::initialize() {
       return false;
     }
     for (auto &Def : MI.all_defs())
-      if (Def.isReg() && Def.getReg().isPhysical())
+      if (Def.isReg() && Def.getReg().isPhysical()) {
+        LLVM_DEBUG(dbgs() << "Physical registers are not supported in "
```

Comment:
Maybe a bit noisy

Reply:
Indeed, there is more debug information. Our consideration is that as a newly proposed software pipelining algorithm, the window algorithm may require more detailed output information to facilitate analysis and debugging.